src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
```

Comment:
```suggestion
     private final Color[] colors = Color.values();
```

Reply:
not fixed

Reply:
I don't really unerstand what I have to fix here